pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -1083,19 +1079,13 @@ void Skip2NextTokenOccurrence(int target) :
     */
     specToken = nextToken.specialToken;
     if (null!= specToken)
-    {
```

Comment:
I'd rather keep these braces. What happens if someone later wants to add a second statement and forgets to put it into the braces?